readme.md

```diff
@@ -102,7 +102,13 @@ If you have an existing S3 bucket, you can use it. If not, used the following st
 aws s3 mb s3://<bucket_name> --region <aws_region>
 ```
 
-### 4. Create EKS Secret to store user credentials
+### 4. Create EKS Volume Snapshot Classes and Secret to store user credentials
```

Comment:
@promiseowolabi Why did you add that? The Snapshot class is created as part of the terraform script.

Reply:
Interesting, it wasn't installed for me, so I had to create them manually. It might be due to terraform failing to get the add-on. See email I sent you.

Reply:
lets resolve that separately. remove the change from the PR and submit it again